protobuf/v1/replica.proto

```diff
@@ -15,6 +15,60 @@ service ReplicaRpc {
   rpc ListReplicas (ListReplicaOptions) returns (ListReplicasResponse) {}
   rpc ShareReplica (ShareReplicaRequest) returns (Replica) {}
   rpc UnshareReplica (UnshareReplicaRequest) returns (Replica) {}
+
+  // Wipe replica
+  rpc WipeReplica (WipeReplicaRequest) returns (stream WipeReplicaResponse) {}
+}
+
+message WipeReplicaRequest {
+  // Uuid of the replica.
+  string                     uuid = 1;
+  oneof pool {
+    // Name of the pool where the replica resides.
+    string              pool_name = 2;
+    // Optional uuid for the pool where the replica resides.
+    string              pool_uuid = 3;
+  }
+  StreamWipeOptions  wipe_options = 4;
+}
+
+message WipeOptions {
+  enum WipeMethod {
+    // Don't actually wipe, just pretend.
+    NONE                        = 0;
+    // Wipe by writing zeroes.
+    WRITE_ZEROES                = 1;
+    // Wipe by sending unmap/trim.
+    UNMAP                       = 2;
+    // Wipe by writing a given patter (see write_pattern(6) field).
+    WRITE_PATTERN               = 3;
+  }
+  // Method used to wipe the bdev.
+  WipeMethod        wipe_method = 5;
+  // When using WRITE_PATTERN, wipe using this 32bit write pattern
+  // Default: 0xDEADBEEF.
+  optional uint32 write_pattern = 6;
+}
+
+message StreamWipeOptions {
+  WipeOptions       options = 1;
+  // Notify client when every chunk is complete.
+  uint64         chunk_size = 4;
```

Comment:
Any specific reason We have made the sequence id for `chunk_size` to 4 ? Logically it should be 2 ?
Also for `wipe_method` and `write_pattern` should be 1 and 2 respectively ?

Reply:
The reason is my mistake :D

Reply:
Done